client/src/modules/multiple_payroll/multiple_payroll.service.js

```diff
@@ -94,9 +94,16 @@ function MultiplePayrollService(
   }
 
   // downloads a type of report based on the
-  function download(type) {
+  function download(type, getSelectedEmployes) {
```

Comment:
Same typo here - `Employes` instead of `Employees`.

Could you rename this `employeeList` or something like that?  `getSelectedEmployes` is the same name you gave the function in the controller, so I think it was a function at first.